source/_components/light.ads.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: page
+title: "ADS Light"
+description: "Instruction on how to set up ADS lights within Home Assistant."
+date: 2017-10-25 10:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Light
+ha_release: 0.56
+ha_iot_class: "Local Push"
+---
+
+To use your ADS device, you first have to set up your [ADS
+hub](/components/ads/) and then add the following to your `configuration.yaml`
+file:
+
+```yaml
+# Example configuration.yaml entry
+light:
+  - platform: ads
+    adsvar: GVL.enable_light
+    adsvar_brightness: GVL.brightness
+```
+
+Configuration variables:
+
+- **adsvar** (*Required*): The name of the boolean variable that switches the
```

Comment:
Please use the configuration tags for listing configuration variables. For more information:
https://home-assistant.io/developers/documentation/create_page/#configuration